src/server/migrations/0.8.0-1.0.0/sql/readings/drop_old_views.sql

```diff
@@ -0,0 +1,6 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+DROP VIEW IF EXISTS daily_readings;
+DROP VIEW IF EXISTS hourly_readings;
```

Comment:
I test this command manually and get the following error:
`ERROR:  "daily_readings" is not a view
HINT:  Use DROP MATERIALIZED VIEW to remove a materialized view.`
Do you think we should change this into `DROP MATERIALIZED VIEW`?